docs/Presentations/OSR-Roundtable.md

```diff
@@ -0,0 +1,102 @@
+
+## Compliance Cow
+
+- Compliance Cow is an open-source tool designed to automate and simplify the process of ensuring compliance with open-source licenses and organizational policies. It helps organizations maintain visibility over their software dependencies and verify that they adhere to legal and policy requirements. The tool is particularly useful for industries like finance, where compliance with strict regulatory standards is crucial. Key features include:
+- License Tracking and Enforcement: Compliance Cow can scan software repositories to detect and validate open-source licenses, ensuring that the right licenses are used and that no incompatible licenses are inadvertently introduced into the codebase.
+- Automated Compliance Audits: The tool enables continuous compliance monitoring by automatically scanning code during the development lifecycle and flagging any non-compliant or risky components.
+- Integration with CI/CD Pipelines: It integrates seamlessly into CI/CD workflows, allowing teams to address compliance issues early in the software development process, avoiding costly remediation later on
+
+By offering automated checks and a detailed view of OSS license obligations, Compliance Cow allows financial institutions to confidently use and contribute to open-source software while ensuring they remain compliant with legal and regulatory frameworks.
+
+## GitProxy
+
+GitProxy is another open-source security tool introduced at the forum, designed to provide secure access to open-source repositories in a controlled and compliant manner. GitProxy acts as a security layer between internal corporate networks and external Git repositories (e.g., GitHub, GitLab), protecting organizations from potential security risks associated with pulling unvetted or malicious code from public repositories.
+
+- Secure Repository Access: GitProxy allows organizations to manage which external Git repositories their developers can access, ensuring that all code pulled into the organization meets security and compliance standards.
+- Code Vetting and Whitelisting: It can be configured to vet code from open-source repositories, automatically rejecting or flagging repositories that don’t meet certain security criteria or are from unknown sources.
+- Audit Trails and Compliance Logging: GitProxy provides detailed audit trails of repository access and code downloads, ensuring that all open- source usage is logged for compliance purposes. This feature is especially important for regulatory reporting and audit requirements. Integration with Internal Policies: It enforces organizational policies on open-source contributions, ensuring that only approved code is pushed to public repositories from the company’s developers
+
+# Reference Links:
+
+https://www.finos.org/blog/finos-q4-and-osff-announcements 
```

Comment:
I would suggest adding * to these two lines, as now they show up in the same line.